### DISCUSSION TITLE
[Unstable] SelectAlways EventSet can get selected even its Commonness is 0

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#14799] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

I make an additive eventset that will always trigger in outpost (additive="true" selectalways="true")
but I dont want it to be selected in endzone (endoutpostmid and endoutpostfinal are outposts)
so I'm doing this
<Override leveltype="endoutpostmid" commonness="0" />
<Override leveltype="endoutpostfinal" commonness="0" />
but the event still triggers because selectalways="true"

### Reproduction steps

1. make two selectalways additive eventsets: test1, test2
2. try to make test1 get selected in endoutpostmid and test2 get selected in endoutpostfinal
3. both eventset will be selected because selectalways="true"

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.6.4.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by A2ur3** [comment]
```xml
<EventSet identifier="test_outpostevents" leveltype="outpost" allowatstart="true" chooserandom="false" ignorecooldown="true" additive="true" selectalways="true">
  <Commonness commonness="1">
    <Override leveltype="endoutpostmid" commonness="0" />
    <Override leveltype="endoutpostfinal" commonness="0" />
  </Commonness>
  <ScriptedEvent identifier="testevent" />
</EventSet>

**Comment by Regalis11** [comment]
This has now been addressed in our private repo, the fix is included in the latest unstable build and coming in the next release.
